### PR TITLE
add Homepage

### PR DESCRIPTION
**Pull request by lauravikanis**
#14 
Add a Homepage which shows 3 Cards
Also added a whole bunch of assets which are going to be used later
=> Commit smaller commits more often

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import styled from "styled-components";
+import Card from "../components/Card/Card";
+import tourPath from "../assets/tour.svg";
+import searchPath from "../assets/search.svg";
+import favoritePath from "../assets/favourites.svg";
+
+const HomeDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  /* justify-content: center; */
```
Comment (kschmitz123) [inline client/src/pages/Home.js:11]:
```suggestion

```

**Review by boldCM — APPROVED** [review]
lgtm🎉

File: client/src/components/Card/Card.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
```
Comment (macebm) [inline client/src/components/Card/Card.js:1]: Maybe use styled-components/macro

**Comment by BelaM80** [comment]
lgtm🎉

**Pull request merged**